### ISSUE TITLE
geom_blank() throws error for non-cartesian coordinates

### ISSUE THREAD

**Issue by cpsievert** [hadley/ggplot2#1248]
```r
df <- data.frame(x1 = c(1, 10), y1 = c(1, 5))
p <- ggplot(df, aes(x1, y1)) + geom_blank() 
```

```r
p #works
p + coord_trans(ytrans = "log10") 
#> Error in log(x, base) : non-numeric argument to mathematical function
p + coord_polar() 
#> Error in if (ends_apart < 0.05) { : argument is of length zero
```

**Comment by hadley** [comment]
Are you sure you have the latest dev version? That works for me

**Comment by hadley** [comment]
Oh, I think I incidentally fixed this about 15 minutes ago in 1ab70def

**Comment by cpsievert** [comment]
Can confirm. @hadley is operating at warp speed :stuck_out_tongue_winking_eye: :+1:
